products/ocp4/profiles/stig.profile

```diff
@@ -0,0 +1,21 @@
+documentation_complete: true
+
+platform: ocp4
+
+metadata:
+    SMEs:
+        - jhrozek
+        - Vincent056
+        - mrogers950
+        - rhmdnd
+        - david-rh
+
+reference: https://dl.dod.cyber.mil/wp-content/uploads/stigs/zip/U_Container_Platform_V1R3_SRG.zip
+
+title: '[DRAFT] DISA STIG for Red Hat OpenShift Container Platform 4'
+
+description: |-
+    This is a draft profile for experimental purposes. It is not based on the DISA STIG for OCP4, because one was not available at the time yet. This profile contains configuration checks that align to the DISA STIG for Red Hat OpenShift Container Platform 4.
```

Comment:
Sorry, one more nit (my fault), can you keep the text within ~80 columns? (`!fmt -80` in vim does the trick for me)